common.js

```diff
@@ -166,6 +166,11 @@ var ccg = {
       ],
       date: "2011",
       publisher: "Information and Privacy Commissioner"
+    },
+    "IPFS": {
+      title: "InterPlanetary File System (IPFS)",
+      href: "https://docs.ipfs.io/",
+      publisher: "Protocol Labs"
```

Comment:
```suggestion
    },
    "IPFS": {
      title: "InterPlanetary File System (IPFS)",
      href: "https://docs.ipfs.io/",
      publisher: "Protocol Labs"
```

No longer needed since the only reference in the spec doesn't exist anymore.